tests/test_admin.py

```diff
@@ -28,6 +28,7 @@
 from flask_admin import Admin, menu
 from invenio_admin import InvenioAdmin
 from invenio_db import db
+from six import b
```

Comment:
Yes, I need it to make this line compatible with Python 3:
`assert b'This field is required.' in res.data`